File: bd_zoo/frontend/src/component/zooStorage/Delivery.js

const Delivery = (props) => {

    return (
        <div className="delivery">
            <h2>{props.delivery.foodType}: </h2>
            <p>Колличество - {props.delivery.count}</p>
            <p>Цена - {props.delivery.price}</p>
            <p>Дата - {props.delivery.date}</p>
        </div>
    );
};

export default Delivery;

File: bd_zoo/backend/src/main/java/com/pankov/bd_zoo/component/delivery/Delivery.java
package com.pankov.bd_zoo.component.delivery;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.pankov.bd_zoo.component.food.Food;
import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDate;


@Entity
@Table(name = "deliveries")
@Data
public class Delivery {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "deliver_name")
    private String name;

    @Column(name = "count", nullable = false)
    private Integer count;

    @Column(name = "price", nullable = false)
    private Integer price;

    @Column(name = "date", nullable = false)
    private LocalDate date;

    @JsonIgnore
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "food_id")
    private Food food;
}
